src/backend/drivers/cswordbiblemoduleinfo.cpp

```diff
@@ -42,19 +42,19 @@ void CSwordBibleModuleInfo::initBounds() const {
     const bool oldStatus = m.isSkipConsecutiveLinks();
     m.setSkipConsecutiveLinks(true);
 
+    sword::VerseKey * key = static_cast<sword::VerseKey *>(m.getKey());
```

Comment:
Since pointer `key` is no longer changed, it can be marked `const`, i.e. `sword::VerseKey * const key`